### PR TITLE
Major UI refactor and code improvements

### PR DESCRIPTION
**Pull request by steveski**
1. The screen layout was completely out on non standard aspect ratios such as 49" ultra wide monitors where the fields were rendered far left of the png form. I've removed the background pngs completely and now building the forms for sell and buy using html and css. Also made an SVG logo to try to make the forms look close to the original pngs.
2. Business name for the form title is now set in the config file.
3. Config file "Occasions" coordinate now converted to a table containing a vector which makes it cleaner to read and edit as well as a little less code to use these coords throughout the resource.
4. Removed jQuery and using Vue.js to leverage two way model binding which makes the coding simpler to integrate with th UI. It's alot cleaner now to make changes ongoing.
5. The entire charinfo object returned from QB was being passed to the form. The system now only sends to fields from charinfo that are required by the web form.
6. When selling direct to the dealer a confirmation is displayed above the to ensure the palyer really wants to sell. Driving away from the zone will make it cancel. A thread is being created for the message and the deletion but once "cancel" is selected the thread completes to free up resources.

### CODE REVIEW

**Comment by GhzGarage** [comment]
Looks good except for the author line in the manifest

File: fxmanifest.lua [hunk 1]
```diff
@@ -1,8 +1,9 @@
 fx_version 'cerulean'
 game 'gta5'
 
+author 'Original author unknown. Changes by Harry The Bastard'
```
Comment (Newtonzz) [inline fxmanifest.lua:4]: No

**Review by Newtonzz — CHANGES_REQUESTED** [review]
why even bother changing the manifest file?

**Comment by steveski** [comment]
> why even bother changing the manifest file?

Because there are new files to include and some removed

File: fxmanifest.lua (hunk 1 above)
Comment (steveski) [reply fxmanifest.lua:4]: What is the issue specifically, my name or the presence of the entry?

Comment (steveski) [reply fxmanifest.lua]: I've made the assumption that the line should be completely removed.

**Pull request merged**